frontend/src/components/Post/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import moment from 'moment'
+
+const Post = ({ index, date, title, url, sourceName, sourceURL, categories = [] }) => (
+  <li>
+    <div className="title">
+      <a href={url} target="_blank">
+        {title}
+      </a>
+    </div>
+    <div className="about">
+      <span>By </span>
+      <a href={sourceURL} target="_blank">{sourceName}</a>
+      &nbsp;
+      <span>{moment(new Date(date)).fromNow()}</span>
```

Comment:
moment.js is a very heavy lib... we could probably use [dayjs](https://github.com/xx45/dayjs#difference) and calculate if we should display "few moments ago"/"last hour"/"X hours ago"/"X days ago"

Reply:
hahahahahaha... I'll remove it, and use pure JS.